www/layouts/partials/header.html

```diff
@@ -1,7 +1,10 @@
 <header id="main">
   <div class="row middle-xs">
     <div class="col-sm-3 col-xs-7 logo-wrapper">
-      <a href="/"><img src="https://materialize.io/wp-content/themes/materialize/img/logo.svg" alt="Materialize Logo">
+      <a href="/"><img class=" lazyloaded"
+        src="https://materialize.io/wp-content/uploads/2020/02/materialize_logo_primary-1.png"
+        data-src="https://materialize.io/wp-content/uploads/2020/02/materialize_logo_primary-1.png"
```

Comment:
What's up with this `lazyloaded` class and the `data-src` attribute?

Reply:
I've just been copying verbatims from the home page to make it easier to keep these things in sync with further copy/paste. I agree they're odd but don't really want to open the can of worms of upstream fixes.